app/javascript/src/components/Dashboard/Notes/constants.js

```diff
@@ -3,11 +3,66 @@ import * as yup from "yup";
 export const NOTES_FORM_INITIAL_FORM_VALUES = {
   title: "",
   description: "",
+  contact: "",
+  tag: "",
 };
 
+export const ASSIGNED_CONTACT = [
+  {
+    label: "Admin",
+    value: "admin",
+  },
+  {
+    label: "Non Admin",
+    value: "non-admin",
+  },
+];
+
+export const TAGS = [
+  {
+    label: "Getting Started",
+    value: 1,
+  },
+  {
+    label: "Meeting",
+    value: 2,
+  },
+  {
+    label: "Learning",
+    value: 3,
+  },
+  {
+    label: "Onboarding",
+    value: 4,
+  },
+  {
+    label: "Leaves and Holidays",
+    value: 5,
+  },
+];
+
 export const NOTES_FORM_VALIDATION_SCHEMA = yup.object().shape({
-  title: yup.string().required("Title is required"),
-  description: yup.string().required("Description is required"),
+  title: yup.string().required("Title is required."),
+  description: yup.string().required("Description is required."),
+  contact: yup
+    .object({
+      label: yup.string().oneOf(ASSIGNED_CONTACT.map(contact => contact.label)),
+      value: yup.string().oneOf(ASSIGNED_CONTACT.map(contact => contact.value)),
+    })
+    .nullable()
+    .required("Assigned contact is required."),
+  tag: yup
+    .array(
```

Comment:
Here your validation suggests that this entity is an array, meaning multiple values. But you have named this as singular `tag`. Doesn't make sense to me.